learn/getting_started/quick_start.md

```diff
@@ -53,11 +53,13 @@ These commands launch the **latest stable release** of Meilisearch.
 # Fetch the latest version of Meilisearch image from DockerHub
 docker pull getmeili/meilisearch:v0.27.1
 
-# Launch Meilisearch
+# Launch Meilisearch with a master key and deactivate analytics
 docker run -it --rm \
     -p 7700:7700 \
+    -e MEILI_MASTER_KEY='myMasterKey'\
     -v $(pwd)/meili_data:/meili_data \
     getmeili/meilisearch:v0.27.1
+    meilisearch --no-analytics
```

Comment:
This change might result in us losing some data from Docker users. How about we replace `--no-analytics` with `--env="development"`?

Also, have you tested this to be sure that it works?

Reply:
Yes I did